src/common/components/paymentMethods/paymentMethodForm/paymentMethodForm.modal.tpl.html

```diff
@@ -1,19 +1,43 @@
-<div class="modal-body">
-  <div class="row">
-    <div class="col-sm-12">
-      <h3 class="text-center border-bottom-small">
-        <span ng-if="!$ctrl.resolve.paymentMethod" translate>Add New Payment Method</span>
-        <span ng-if="$ctrl.resolve.paymentMethod" translate>Edit Payment Method</span>
-      </h3>
-      <payment-method-form submitted="$ctrl.submitted" on-submit="$ctrl.onSubmit(success, data)" submission-error="$ctrl.resolve.submissionError" payment-method="$ctrl.resolve.paymentMethod" mailing-address="$ctrl.resolve.mailingAddress"></payment-method-form>
+<loading-overlay ng-if="$ctrl.submitted"></loading-overlay>
```

Comment:
This somehow still isn't right. If I hit submit with an invalid form, the loading indicator takes over and submitted never gets set back to false. Let me know if you want me to take a look at it.